bin/cli.js

```diff
@@ -50,21 +50,51 @@ const args = yargs
     type: 'string',
     describe: 'custom config file',
   })
+  .option('no-config', {
+    type: 'boolean',
+    describe: 'do not look for a configuration file',
+  })
   .argv;
 
+const loadFiles = files => {
+  const result = [];
+  files
+    .map(file => {
+      if (fs.existsSync(file + '.js')) {
+        return file + '.js';
+      } else if (fs.existsSync(file)) {
+        return file;
+      } else {
+        console.error('File does not exist:', file);
+        process.exit(1);
+        return '';
+      }
+    })
+    .forEach(file => {
+      if (fs.statSync(file).isDirectory()) {
+        const subfiles = fs.readdirSync(file).map(f => path.join(file, f));
+        result.push(...loadFiles(subfiles));
+      } else if (common.fileExt(file) === 'js') {
+        result.push(file);
+      }
+    });
+  return result;
+};
+
 const getConfig = args => {
   let cfg = {};
-  if (args.config && fs.existsSync(args.config)) {
-    cfg = JSON.parse(fs.readFileSync(args.config));
-  } else if (args.config === '') {
-    cfg = JSON.parse(fs.readFileSync('.metadocrc'));
+  if (args.config) {
```

Comment:
Will this use '.metadocrc' for `metadoc test`?

Reply:
`metadoc` will use `.metadocrc` unless you specify `--no-config` flag or another config file with `--config`.

Reply:
I've just run `./bin/cli.js` on this branch and got `No files were specified. Exiting...` even though we have `.metadocrc` in the package root folder.

Reply:
Fixed in https://github.com/metarhia/metadoc/pull/22/commits/c457b2fbd71465f209412a91d5833e66597aeac9.